tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+*** Settings ***
+Library         OpenShiftCLI
+Library         OpenShiftLibrary
+Resource        ../../Resources/Common.robot
+Resource        ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+
+
+*** Test Cases ***
+Verify If Unauthorisez User Can Not Change The Permission
+    [Tags]  ODS-1661
+    ...     ODS-1660
+    Set Library Search Order  SeleniumLibrary
+    Check User Management Option Is Available For The User   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Clear All The Input Values In User Managermant
+    Add OpenShift Groups To Data Science Administrators         cluster-admins  rhods-admins
+    Add OpenShift Users groups to Data Science administrators   system:authenticated
+    Save Changes In User Management Setting
+
+    Verify Expect Changes In AdminGroups Are Present In CRD     cluster-admins  rhods-admins
```

Comment:
hm it's not a good idea to use cluster-admins. I mean, it's good for first testing, but the in the finale working feature, cluster admins will automatically get the Admin access. So, pls use a different group

Reply:
i agree will change those